Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -8065,9 +8086,8 @@
 				A757E9EF1D19C37F00A5C978 /* ActivitySurveyResponseCell.swift in Sources */,
 				D63BBCDA217E7802007E01F0 /* CreditCardCell.swift in Sources */,
 				473632A426EBAC6A001B0D39 /* RiskMessagingViewController.swift in Sources */,
-				191E601F2A93F318001413B2 /* ProjectTabQuestionAnswerCell.swift in Sources */,
 				191E601F2A93F318001413B2 /* ProjectTabAIGenerationCell.swift in Sources */,
-				018422BE1D2C484700CA7566 /* DashboardTitleView.swift in Sources */,
+				191E601F2A93F318001413B2 /* ProjectTabAIGenerationCell.swift in Sources */,
```

Comment:
I wonder what caused these changes? maybe your branch is out of sync with `main`?